tests/e2e/specs/wcpay/merchant/merchant-orders-partial-refund.spec.js

```diff
@@ -48,167 +48,153 @@ const dataTable = [
-					config.get( 'addresses.customer.billing' ),
-					lineItems
-				);
-				await fillCardDetails( page, card );
-				await shopper.placeOrder();
-				await expect( page ).toMatch( 'Order received' );
-
-				// Remember the order ID and order total. We will need them later.
-				orderId = await page.$eval(
-					'.woocommerce-order-overview__order.order > strong',
-					( el ) => el.innerText
-				);
-				orderTotal = await page.$eval(
-					'.woocommerce-order-overview__total strong',
-					( el ) => Number( el.innerText.replace( '$', '' ) )
-				);
-
-				// Login as merchant
-				await merchant.login();
-			}, 200000 );
-
-			afterAll( async () => {
-				await merchant.logout();
-			} );
-
-			it( `should refund ${ testName }`, async () => {
-				const refundReason = `Refunding ${ testName }`;
-				const refundTotal = refundInputs
-					.map( ( { refundAmount } ) => refundAmount )
-					.reduce( ( acc, cur ) => acc + cur );
-				const refundTotalString = refundTotal.toFixed( 2 );
-				const netPayment = ( orderTotal - refundTotal ).toFixed( 2 );
-
-				await merchant.goToOrder( orderId );
-
-				// We need to remove any listeners on the `dialog` event otherwise we can't catch the dialog below
-				await page.removeAllListeners( 'dialog' );
-
-				// Click the Refund button
-				await expect( page ).toClick( 'button.refund-items' );
-
-				// Fill up the quantity and/or amount to be refunded per line item
-				const rows = await page.$$( '#order_line_items tr.item' );
-				for ( let i = 0; i < refundInputs.length; i++ ) {
-					const { refundQty, refundAmount } = refundInputs[ i ];
-					const row = rows[ i ];
-
-					if ( refundQty ) {
-						await expect( row ).toFill(
-							'.refund_order_item_qty',
-							`${ refundQty }`
-						);
-					} else {
-						await expect( row ).toFill(
-							'.refund_line_total',
-							`${ refundAmount }`
-						);
-					}
+describe.each( dataTable )(
+	'Order > Partial refund',
+	( testName, { lineItems, refundInputs } ) => {
+		let orderId;
+		let orderTotal;
+
+		beforeAll( async () => {
+			// Disable multi currency in the merchant settings.
+			await merchant.login();
+			await merchantWCP.deactivateMulticurrency();
+			await merchant.logout();
+
+			// Set up the test order
+			await setupProductCheckout(
+				config.get( 'addresses.customer.billing' ),
+				lineItems
+			);
+			await fillCardDetails( page, card );
+			await shopper.placeOrder();
+			await expect( page ).toMatch( 'Order received' );
+
+			// Remember the order ID and order total. We will need them later.
+			orderId = await page.$eval(
+				'.woocommerce-order-overview__order.order > strong',
+				( el ) => el.innerText
+			);
+			orderTotal = await page.$eval(
+				'.woocommerce-order-overview__total strong',
+				( el ) => Number( el.innerText.replace( '$', '' ) )
+			);
+
+			// Login as merchant
+			await merchant.login();
+		}, 200000 );
+
+		afterAll( async () => {
+			await merchantWCP.activateMulticurrency();
```

Comment:
I don't think we need it. We should drop this call.

Reply:
@kalessil To prevent cascade failures, it would be a good idea to reactivate Multi currency once the test is done.

Reply:
@achyuthajoy, good point. @mgascam do we know if multi-currency is enabled by default?

Reply:
>  @mgascam do we know if multi-currency is enabled by default?

@kalessil I think so, when I go to Payments > Settings of the client docker instance used by e2e's the Enable Multi-Currency checkbox is active.

Reply:
Oki-doki, approving then.